Testing/functional/tests/cypress/integration/ui/errors/tooManyRequestsAlerts.js

```diff
@@ -0,0 +1,495 @@
+            .click();
+        cy.get("[data-testid=save-as-pdf-dropdown-item]")
+            .should("be.visible")
+            .click();
+        cy.get("[data-testid=genericMessageModal]").should("be.visible");
+        cy.get("[data-testid=genericMessageSubmitBtn]").click();
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+        cy.get("[data-testid=loadingSpinner]").should("not.be.visible");
+    });
+});
+
+describe("Landing Page - Too Many Requests", () => {
+    it("Too Many Requests Banner Appears on 429 Response", () => {
+        cy.intercept("GET", "**/Communication/*", { statusCode: 429 });
+        cy.visit("/");
+
+        cy.contains(
+            "[data-testid=communicationBanner]",
+            "higher than usual site traffic"
+        ).should("be.visible");
+    });
+
+    it("Too Many Requests Banner Doesn't Appear on 200 Response", () => {
+        cy.intercept("GET", "**/Communication/*", { statusCode: 200 }).as(
+            "getCommunication"
+        );
+        cy.visit("/");
+
+        // wait for both Communication calls to complete
+        cy.wait("@getCommunication");
+        cy.wait("@getCommunication");
+
+        cy.contains(
+            "[data-testid=communicationBanner]",
+            "higher than usual site traffic"
+        ).should("not.exist");
+    });
+});
+
+describe("Immunization", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/Immunization?*", {
+            statusCode: 429,
+        });
+        cy.enableModules("Immunization");
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Medication Request", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.enableModules("MedicationRequest");
+        cy.intercept("GET", "**/MedicationRequest/*", {
+            fixture: "MedicationService/medicationRequest.json",
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.intercept("GET", "**/MedicationRequest/*", {
+            statusCode: 429,
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Mobile - COVID-19 Orders", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/Laboratory/Covid19Orders*", {
+            fixture: "LaboratoryService/covid19Orders.json",
```

Comment:
Do we need to check if Covid19Orders are successful before checking Covid19Orders again for 429 on line 276?